src/System.Data.SqlClient/tests/ManualTests/SQL/RandomStressTest/Randomizer.cs

```diff
@@ -256,8 +256,8 @@ public int NextSeed()
             return Next();
         }
 
-        #region Random core implementation taken from .Net 4.0 source code- DO NOT CHANGE BEYOND THIS POINT
-        // The code below was copied from .Net 4.0 sources of Random class
+        #region DO NOT CHANGE BEYOND THIS POINT
+        // The code below was copied from https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Random.cs
```

Comment:
This link will be stale if Random.cs ever moves / anything in its parent hierarchy ever changes (e.g. if "mscorlib" were renamed at some point to "System.Private.CoreLib").  Not sure if that matters.  Only somewhat related, the implementation no longer matches the one in coreclr (do we really need this copy here? cc: @saurabh500)